portal/src/locale-data/en.json

```diff
@@ -183,6 +184,16 @@
   "AddRolesScreen.roleName.description": "A display name of this role in the portal.",
   "AddRolesScreen.roleDescription.title": "Description",
 
+  "RolesList.column.name": "Name",
+  "RolesList.column.key": "Role Key",
+  "RolesList.column.description": "Description",
+  "RolesList.column.action": "Action",
+  "RolesList.delete-role": "Delete Role",
+
+  "DeleteRoleDialog.title": "Delete Role",
+  "DeleteRoleDialog.description": "Are you sure you want to delete the role \"Human Resources Director\"?",
```

Comment:
The role name shouldn't be hard coded.
I've fixed it in https://github.com/authgear/authgear-server/pull/3946